src/bend.ts

```diff
@@ -96,31 +94,20 @@ export class Bend extends Modifier {
    *     width: 8;
    *   }]
    * ```
-   * @param text text for bend ("Full", "Half", etc.) (DEPRECATED)
-   * @param release if true, render a release. (DEPRECATED)
-   * @param phrase if set, ignore "text" and "release", and use the more sophisticated phrase specified
    */
```

Comment:
Oh I see now. They were marked DEPRECATED at some point. Sure, I guess it's fine to remove these constructor params in V5. Interesting that we kept on using the deprecated API in our tests.

Reply:
Looks like it was marked DEPRECATED before 3.0.9, so it has been deprecated for a while.

Removing it now sounds good to me.